### PR TITLE
update description of "Nest an anchor element in a paragraph" as discussed in #38507

### LINKED ISSUE #38507: change challenge "Nest an anchor element in a paragraph" description to avoid users confusion
After a long extenuating discussion (not yet resolved) in which an user refuses to change the existing anchor element on line 4 because challenge description says 

> Now nest the existing a element within a new p element (just after the existing main element).

can it be changed to say "just after the existing opening main tag"?

https://www.freecodecamp.org/learn/responsive-web-design/basic-html-and-html5/nest-an-anchor-element-within-a-paragraph

### PR DESCRIPTION
**Pull request by majestic-owl448**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #38507 

As discussed in #38507 , I'm removing the phrase in parenthesis. It seems the more agreed upon way, and the less disrupting for translations.

### CODE REVIEW

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/nest-an-anchor-element-within-a-paragraph.english.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ The final output of the example will look like this:<br><p>Here's a <a target="_
 ## Instructions
 <section id='instructions'>
 
-Now nest the existing <code>a</code> element within a new <code>p</code> element (just after the existing <code>main</code> element). The new paragraph should have text that says "View more cat photos", where "cat photos" is a link, and the rest of the text is plain text.
+Now nest the existing <code>a</code> element within a new <code>p</code> element. The new paragraph should have text that says "View more cat photos", where "cat photos" is a link, and the rest of the text is plain text.
```
Comment (thecodingaviator) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/nest-an-anchor-element-within-a-paragraph.english.md:32]:
```suggestion
Nest the existing <code>a</code> element within a new <code>p</code> element. The new paragraph should have text that says "View more cat photos", where "cat photos" is a link, and the rest is plain text.
```
Edited for brevity.

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

**Comment by moT01** [comment]
I have blocked this for now. See the [pinned code freeze issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/38308)

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍

**Review by erictleung — APPROVED** [review]

**Review by thecodingaviator — APPROVED** [review]

**Review by thecodingaviator — APPROVED** [review]

**Comment by thecodingaviator** [comment]
Whoops my bad. Double reviewed it. Didn't see it was a blocked one and that I had already been here.

**Review by moT01 — APPROVED** [review]
Thanks for fixing this up @Ieahleen 🎉

**Pull request merged**